sunbeam-python/sunbeam/commands/juju.py

```diff
@@ -1297,6 +1382,26 @@ def run(self, status: Optional["Status"] = None) -> Result:
         return Result(ResultType.COMPLETED)
 
 
+class SaveJujuRemoteUserLocallyStep(SaveJujuUserLocallyStep):
+    """Save remote juju user locally in accounts yaml file."""
+
+    def __init__(self, controller: str, data_location: Path):
+        super().__init__(None, data_location)
```

Comment:
This None will be put into `Saving machine user {name} for local usage` which is weird

Reply:
Fixed in subsequent commit